packages/rancher-cis-benchmark/charts/Chart.yaml

```diff
@@ -1,9 +1,9 @@
 apiVersion: v1
-appVersion: v0.0.1
+appVersion: v0.2.0
 description: The cis-operator enables running CIS benchmark security scans on a kubernetes
   cluster
 name: rancher-cis-benchmark
-version: 0.0.1
+version: 0.2.000
```

Comment:
Swapping this for 1.0.000 now

Reply:
Swapped to 1.0.000  and also added a logo link and the logo image is added to dev-2.5 PR - is that correct @cbron